spec/compiler/crystal/tools/context_spec.cr

```diff
@@ -1,26 +1,24 @@
 require "spec"
 require "yaml"
-require "../../../../src/compiler/crystal/**"
```

Comment:
I'm pretty sure that's incorrect. The `bin/crystal` wrapper should edit the load path so that any part of the stdlib is loaded from the git repo, not the distribution. There's nothing special about the compiler requires.